### PR TITLE
Improve performance of data generation

### PR DESCRIPTION
**Pull request by wendigo**
Continuation of https://github.com/trinodb/tpch/pull/34

### CODE REVIEW

File: src/main/java/io/trino/tpch/GenerateUtils.java [hunk 1]
```diff
@@ -151,7 +151,9 @@ private static boolean isLeapYear(int year)
 
     public static String formatMoney(long value)
     {
-        // todo there must be a better way to do this
-        return new BigDecimal(value).divide(new BigDecimal(100)).setScale(2).toString();
+        long unsignedValue = abs(value);
+        long significant = unsignedValue / 100;
+        long remainder = unsignedValue - significant * 100;
```
Comment (martint) [inline src/main/java/io/trino/tpch/GenerateUtils.java:156]: Why not just use `%`?

File: src/main/java/io/trino/tpch/GenerateUtils.java [hunk 2]
```diff
@@ -151,7 +151,9 @@ private static boolean isLeapYear(int year)
 
     public static String formatMoney(long value)
     {
-        // todo there must be a better way to do this
-        return new BigDecimal(value).divide(new BigDecimal(100)).setScale(2).toString();
+        long unsignedValue = abs(value);
```
[the remark this answers is not in the document]
Comment (wendigo) [reply src/main/java/io/trino/tpch/GenerateUtils.java:154]: absExact is JDK15+ and tpch is 11 at the moment

[the remark this answers is not in the document]
Comment (martint) [reply src/main/java/io/trino/tpch/GenerateUtils.java:155]: In that case, add a checkArgument to ensure it’s not that value.

**Comment by wendigo** [comment]
@martint please review & merge :)

**Pull request merged**